pydis_site/apps/api/viewsets/bot/aoc_link.py

```diff
@@ -55,14 +54,14 @@ class AocAccountLinkViewSet(
 
     #### Status codes
     - 204: returned on success
-    - 400: if one of the given fields is invalid
+    - 400: if one of the given fields was invalid
 
     ### DELETE /bot/aoc-account-links/<user__id:int>
     Deletes the AoC account link item with the given `user__id`.
 
     #### Status codes
-    - 204: returned on success
-    - 404: if the AoC account link with the given user__id does not exist
+    - 204: returned on succes
```

Comment:
This was correct originally.
```suggestion
    - 204: returned on success
```